test/naming.dart

```diff
@@ -19,19 +19,27 @@ const String platformBrowser = 'browser';
 const String platformMock = 'mock';
 const String platformVM = 'vm';
 
-/// Integration test group names.
-const String integrationHttpBrowser = '[integration] HTTP ($platformBrowser)';
-const String integrationHttpMock = '[integration] HTTP ($platformMock)';
-const String integrationHttpVM = '[integration] HTTP ($platformVM)';
-const String integrationWebSocketBrowser =
-    '[integration] WS ($platformBrowser)';
-const String integrationWebSocketMock = '[integration] WS ($platformMock)';
-const String integrationWebSocketVM = '[integration] WS ($platformVM)';
+/// Test types.
+const String testTypeIntegration = 'integration';
+const String testTypeUnit = 'unit';
 
-/// Unit test group names.
-const String unitHttpBrowser = '[unit] HTTP ($platformBrowser)';
-const String unitHttpMock = '[unit] HTTP ($platformMock)';
-const String unitHttpVM = '[unit] HTTP ($platformVM)';
-const String unitWebSocketBrowser = '[unit] WS ($platformBrowser)';
-const String unitWebSocketMock = '[unit] WS ($platformMock)';
-const String unitWebSocketVM = '[unit] WS ($platformVM)';
+/// Topics
+const String topicHttp = 'HTTP';
+const String topicMocks = 'Mocks';
+const String topicPlatformAdapter = 'Platform Adapter';
+const String topicWebSocket = 'WS';
+
+class Naming {
+  String platform;
+  String testType;
+  String topic;
+
+  @override
```

Comment:
I was curious if one must include `@override` with the toString function.  It turns out that it's not necessary : https://dartpad.dartlang.org/053c01d956e344df2f67
No biggie including it though.